pkg/apis/kops/v1alpha1/componentconfig.go

```diff
@@ -377,6 +377,8 @@ type KubeAPIServerConfig struct {
 	RuntimeConfig         map[string]string `json:"runtimeConfig,omitempty" flag:"runtime-config"`
 
 	AnonymousAuth *bool `json:"anonymousAuth,omitempty" flag:"anonymous-auth"`
+
+	KubeletPreferredAddressTypes []string `json:"kubeletPreferredAddressTypes,omitempty" flag:"kubelet-preferred-address-types"`
```

Comment:
` pkg/apis/kops/v1alpha1/componentconfig.go` I think we need ` pkg/apis/kops/v1alpha2/componentconfig.go`